guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -574,6 +574,8 @@ You need to read configuration from different sources (like database or filesyst
 
 6.4.3.8. Data objects returned by service contracts SHOULD be fully loaded to ensure consistency.
 
+6.4.3.9. Service contracts SHOULD allow client side generated IDs. A service SHOULD accept an ID instead of generating it. A service SHOULD use `string` as a type for ID, so that it is compatible with common algorithms for client side generated IDs.
```

Comment:
This statement is true for GraphQL schema but not necessary for the service contracts.
In code, client-generated Id, or its particular example UUID could have another datatype than string.